_faq/installation_and_deployment/run-forseti-without-org-access.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Can I install and run Forseti without Org Access?
+order: 6
+---
+{::options auto_ids="false" /}
+
+Although Forseti is designed to install and run out-of-the box with complete
+org access, you can install Forseti if you aren't an org admin. You'll then
+manually give Forseti the permissions to inventory and audit a subset of
+resources by one specific folder or by projects that are directly under
+the org.
+
+If you aren't an org admin and an org admin isn't available to grant org
+access to Forseti, follow the process below:
+
+1. Run the Forseti installer. The installer will try to assign org-level
+roles. You can ignore the org-level role assignment.
+
+1. The installer will create all the necessary Forseti resources: Forseti
+project, VM instances, Cloud SQL database, and service accounts.
+
+1. To inventory all the resources in a folder:
+
+   1. Edit `forseti_conf_server.yaml` and point the `root_resource_id`
+to the target folder: `folders/<foo_folder_id>`.
+   1. Force the server to reload the updated configuration.
+   1. Grant the folder editor role to the Forseti server service account, on the target folder.
```

Comment:
Link to more info about how to grant roles here and on the next step. For example:

`grant the [folder editor role](link to info)`

Reply:
done